codegen.ml

```diff
@@ -56,15 +56,16 @@ and find_global_class name =
 
 let rec id_gen llbuilder id is_deref =
 	if is_deref then
+(
 		try
-			let _val = Hash.find local_params id in
-			L.build_load _val id llbuilder
+                    (*HH: if build_load function parameters, llvm will yell at you*)
+			Hash.find local_params id
```

Comment:
This is fine since the `local_params` don't need `build_load` so you can remove the HH comment